src/components/ButtonComponent/index.jsx

```diff
@@ -0,0 +1,11 @@
+import * as S from "./ButtonStyled";
+
+export const Button = ({ width, border, color, shadow, background, children }) => {
+  const Click = () => {
+  };
+  return (
+    <div>
+      <S.Button type="button" width={width} border={border} color={color} shadow={shadow} background={background} onClick={Click}>{children}</S.Button>
```

Comment:
**Uma dica**

Quando temos props que só serão repassadas sem nenhum tratamento, utilizamos o spread. 

[Da uma olhadinha aqui](https://medium.com/coding-at-dawn/how-to-pass-all-props-to-a-child-component-in-react-bded9e38bb62) :smiley: